static/docs/tutorials/index.md

```diff
@@ -0,0 +1,70 @@
+# DVC Basic Concepts and Features
+
+Learn basic concepts and features of DVC with interactive lessons:
+
+1. [Data Management](https://katacoda.com/dvc/courses/basics/data) <br/> The
+   core function of DVC is data tracking and management. Let's see how to do it.
+
+2. [Getting the Best Performance](https://katacoda.com/dvc/courses/basics/performance)
+   <br/> It is important to optimize the DVC setup for having the best
+   performance with handling big data files.
+
+3. [Tracking Data Versions](https://katacoda.com/dvc/courses/basics/versioning)
+   <br/> DVC takes advantage of GIT's versioning features to keep track of the
+   data versions.
+
+4. [Sharing Data](https://katacoda.com/dvc/courses/basics/sharing) <br/> DVC
+   facilitates sharing of data between different people that work on the same
+   project.
+
+5. [Stages And Pipelines](https://katacoda.com/dvc/courses/basics/pipelines)
+   <br/> DVC has a built-in way to connect ML steps into a DAG and run the full
+   pipeline end-to-end.
+
+# Interactive Tutorials
+
+1. [Data Versioning](https://katacoda.com/dvc/courses/tutorials/versioning)
+   <br/> Using DVC commands to work with multiple versions of datasets and ML
+   models.
+
+2. [Pipelines](https://katacoda.com/dvc/courses/tutorials/pipelines) <br/> Using
+   DVC commands to build a simple ML pipeline.
+
+3. [dvc fetch](https://katacoda.com/dvc/courses/examples) <br/> We will use an
+   example project with some data, code, ML models, pipeline stages, as well as
+   a few Git tags. Then we will see what happens with dvc fetch as we switch
+   from tag to tag.
+
+# Native Tutorials
+
+1. [Versioning](/docs/tutorials/versioning) <br/> Using DVC commands to work
+   with multiple versions of datasets and ML models.
+
+2. [Pipelines](/docs/tutorials/pipelines) <br/> Using DVC commands to build a
+   simple ML pipeline.
+
+3. [Longer Tutorial](/docs/tutorials/tutorial) <br/> Introduces DVC
+   step-by-step, while additionally explaining in great detail the motivation
+   and what's happening internally.
+
+# Community Blogs and Tutorials
```

Comment:
should be just community tutorials, we should be including here just regular blog posts